Solutions/Fortinet-FortiGate/Playbooks/readme.md

```diff
@@ -39,6 +40,25 @@ You can choose to deploy the whole package: connector + Function App + all three
 
 [![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FFortinet-FortiGate%2Fazuredeploy.json) [![Deploy to Azure](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FFortinet-FortiGate%2Fazuredeploy.json)
 
+## Deploy Custom Connector + 2 Playbook templates
+<a name="onprem">
+
+This package includes:
+* [Custom connector](./CustomConnectorOnPrem)
+* [Function App](../FunctionApp)
+* Three playbook templates leverage fortinet custom connector and Function App:
+  * [Block IP](./Playbooks/PlaybooksOnPrem/Fortinet_ResponseOnIP)
+  * [Block URL](./Playbooks/PlaybooksOnPrem/Fortinet_ResponseOnURL)
+
+
+*The Azure Function handles the Get calls on FortiOS API in the playbook templates. These calls are not part of the custom connector due to platform limitations.*
+
+You can choose to deploy the whole package: connector + Function App + all three playbook templates, or each one seperately from it's specific folder.
+
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FFortinet-FortiGate%2Fazuredeploy.json) [![Deploy to Azure](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FFortinet-FortiGate%2Fazuredeploy.json)
+
+
 # Fortinet connector documentation 
 
 <a name="authentication">
```

Comment:
1. Please correct the spelling 'consloidated'.
2. Change 'fortinet' to 'Fortinet'.